.github/workflows/action-conda-publish.yml

```diff
@@ -51,6 +52,7 @@ jobs:
         run: |
           export BUILD_FOLDER=/tmp/esmvalcore/_build
           # use --label test before actual release and check all works well
+          # esmvalgroup is not the main channel anymore, conda-forge superseded it
```

Comment:
This comment confuses me. Shall we just scrap the whole action?

Reply:
I was thinking about that too - but what if we decide we really need to test something related to the package build (or even more, we want to test the package build) - we already have a working GA workflow rather than fishing around the interwebs for one :grin:

Reply:
We are using this thing called git that should allow us to get it back even in that case :wink:

Reply:
sifting through git history is more painful than finding pictures of your ex-gf's in your photo album, man :rofl: But OK, I can remove it for good if yout hink it's a good idea

Reply:
That depends on how you maintain your history. If you can't use it to find your own work, you are probably doing it wrong. But that is a topic for a different time.

I think we can remove this.